fake/BUILD

```diff
@@ -0,0 +1,19 @@
+# This pacakge exists because
+# But bazel test //... fails with
+# No test targets were found, yet testing was requested
+# So here is a test:
```

Comment:
That sort of feels like a band aid. Maybe better to simply not invoke `bazel test` then?

Reply:
Agree. Reverted.